src/components/Contacts/ContactDetails/ContactDetailsTab/Mailing/ContactDetailsTabMailing.tsx

```diff
@@ -0,0 +1,98 @@
+}));
+
+const ContactDetailsMailingMainContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  marginTop: theme.spacing(2),
+}));
+
+const ContactDetailsMailingIcon = styled(LocationOn)(({ theme }) => ({
+  margin: theme.spacing(1),
+  width: '14px',
+  height: '20px',
+  flexGrow: 1,
+}));
+
+const ContactDetailsMailingTexContainer = styled(Box)(({}) => ({
+  flexGrow: 8,
+}));
+
+const ContactDetailsMailingLabelTextContainer = styled(Box)(({}) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  marginTop: '10px',
+}));
+
+const ContactDetailsMailingLabel = styled(Typography)(({ theme }) => ({
+  color: theme.palette.text.hint,
+  marginRight: '5px',
+}));
+
+interface MailingProp {
+  data: ContactDetailsTabQuery;
+}
+
+export const ContactDetailsTabMailing: React.FC<MailingProp> = ({ data }) => {
+  const { t } = useTranslation();
+  return (
+    <Box>
+      <ContactDetailsMailingTitle variant="h6">
+        {t('Mailing')}
+      </ContactDetailsMailingTitle>
+      <ContactDetailsMailingMainContainer>
+        <ContactDetailsMailingIcon />
+        <ContactDetailsMailingTexContainer>
+          {/* Address Section */}
+          <Typography variant="subtitle1">{data.contact.name}</Typography>
+          <Typography variant="subtitle1">
+            {data.contact.primaryAddress.street}
+          </Typography>
+          <Typography variant="subtitle1">
+            {data.contact.primaryAddress.city +
+              ', ' +
+              data.contact.primaryAddress.state +
+              ' ' +
+              data.contact.primaryAddress.postalCode}
+          </Typography>
+          {/* Show More Section */}
+          <ContactDetailsMailingLabelTextContainer>
+            <Link href="#">
+              <Typography variant="subtitle1">{t('Show More')}</Typography>
+            </Link>
+          </ContactDetailsMailingLabelTextContainer>
+          {/* Greeting Section */}
+          <ContactDetailsMailingLabelTextContainer>
+            <ContactDetailsMailingLabel variant="subtitle1">
+              {t('Greeting')}
+            </ContactDetailsMailingLabel>
+            <Typography variant="subtitle1">{data.contact.greeting}</Typography>
+          </ContactDetailsMailingLabelTextContainer>
+          {/* Newsletter Section */}
+          <ContactDetailsMailingLabelTextContainer>
+            <ContactDetailsMailingLabel variant="subtitle1">
+              {t('Newsletter')}
+            </ContactDetailsMailingLabel>
+            <Typography variant="subtitle1">
+              {data.contact.sendNewsletter}
+            </Typography>
+          </ContactDetailsMailingLabelTextContainer>
+          {/* Magizine Section */}
+          <ContactDetailsMailingLabelTextContainer>
+            <ContactDetailsMailingLabel variant="subtitle1">
+              {t('Magazine')}
+            </ContactDetailsMailingLabel>
+            <Typography variant="subtitle1">Not Implemented</Typography>
```

Comment:
I assume this will be implemented in a future pr? Is the data not available yet?

Reply:
@TheNoodleMoose yes the data is not currently available in graphql.